library/std/src/sys/sgx/abi/usercalls/alloc.rs

```diff
@@ -141,7 +141,7 @@ unsafe impl<T: UserSafeSized> UserSafe for [T] {
 
     /// # Safety
     /// Behavior is undefined if any of these conditions are violated:
-    /// * `ptr` must be [valid] for writes of `size` many bytes, and it must be
+    /// * `ptr` must be [valid]:core::ptr#safety for writes of `size` many bytes, and it must be
     ///   properly aligned.
     ///
```

Comment:
Sorry, I meant like this:
```suggestion
    /// * `ptr` must be [valid] for writes of `size` many bytes, and it must be
    ///   properly aligned.
    ///
    /// [valid]: core::ptr#safety
```
Currently this isn't valid markdown syntax (or rather, it will be rendered as `[valid]:core::ptr#safety`).

Reply:
Sorry about that. I fixed it in the latest commit.